docs/content/en/open_source/installation/running-in-production.md

```diff
@@ -55,7 +65,7 @@ concurrent connections.
 
 ### Celery worker
 
-By default, a single mono-process celery worker is spawned. When storing a large amount of findings, leveraging async functions (like deduplication), or both. Eventually, it is important to adjust these parameters to prevent resource starvation. 
+By default, a single mono-process celery worker is spawned. When storing a large amount of findings, leveraging async functions (like deduplication), or both. Eventually, it is important to adjust these parameters to prevent resource starvation.
```

Comment:
"When storying a large amount of findings, leveraging async function (like deduplication), or both." is a sentence fragment and I am not sure what is meant?

Reply:
I didn't change anything here.... update the text a bit.